lib/collection.js

```diff
@@ -2046,9 +1944,10 @@ const findOneAndUpdateSchema = {
   update: { overrideOnly: true },
   new: { overrideOnly: true },
   remove: { overrideOnly: true },
-  serializeFunctions: { type: 'function' },
+  serializeFunctions: { type: 'boolean' },
   checkKeys: { overrideOnly: true },
-  readPreference: { overrideOnly: true }
+  readPreference: { overrideOnly: true },
+  fsync: { type: 'number' }
```

Comment:
fsync is deprecated, but should we document it anyway if we're already going to add it in the schema?